cns/cniconflist/generator_windows.go

```diff
@@ -0,0 +1,13 @@
+package cniconflist
+
+import "errors"
+
+var errNotImplemented = errors.New("cni conflist generator not implemented on Windows")
```

Comment:
Out of curiosity, why? Seems like it just generates JSON... is it that we don't want that JSON to be consumed by anything on Windows?